### PR TITLE
Add pteropus support

### PR DESCRIPTION
**Pull request by hulahermit**
On branch pteropus
Changes to be committed:
	new file:   keyboards/pteropus/config.h
	new file:   keyboards/pteropus/info.json
	new file:   keyboards/pteropus/keymaps/default/keymap.c
	new file:   keyboards/pteropus/readme.md
	new file:   keyboards/pteropus/rules.mk

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/pteropus/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      *               └───┤   │   │   ├───┘
+      *                   └───┘   └───┘
+      */
+    // [0] = LAYOUT_SPLIT_3x5_3(
+    //     KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                               KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+    //     KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                               KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,
+    //     KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                               KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,
+    //                                KC_LGUI, KC_BSPC, KC_SPC,           KC_SPC,  KC_ENT,  KC_RALT
+    // )
+#define LAYOUT(
```
Comment (zvecr) [inline keyboards/pteropus/keymaps/default/keymap.c:27]: This isnt a valid keymap

File: keyboards/pteropus/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# pteropus
+
+![pteropus](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Chi-Han Yang](https://github.com/hulahermit)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (zvecr) [inline keyboards/pteropus/readme.md:9]: Didnt feel like filling these out?

File: keyboards/pteropus/config.h [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Chi-Han Yang (@hulahermit)
```
Comment (zvecr) [inline keyboards/pteropus/config.h:1]: This file should be removed

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/pteropus/info.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT": {
```
[the remark this answers is not in the document]
Comment (hulahermit) [reply keyboards/pteropus/info.json:27]: Thank you for the advice.

File: keyboards/pteropus/keymaps/default/keymap.c (hunk 1 above)
Comment (hulahermit) [reply keyboards/pteropus/keymaps/default/keymap.c:27]: Still learning everything. I didn't mean to push to main qmk this early. I think I will close pull request and test first.

**Comment by hulahermit** [comment]
Too early to submit a pull request.

**Pull request closed without merging**